src/herbivoresAndCarnivores.js

```diff
@@ -1,19 +1,42 @@
-'use strict';
-
-class Animal {
-  // write your code here
-}
-
-class Herbivore extends Animal {
-  // write your code here
-}
-
-class Carnivore extends Animal {
-  // write your code here
-}
-
-module.exports = {
-  Animal,
-  Herbivore,
-  Carnivore,
-};
+'use strict';
+
+class Animal {
+  constructor(name) {
+    Animal.alive.push(this);
+
+    this.name = name;
+    this.health = 100;
+  }
+}
+
+Animal.alive = [];
+
+class Herbivore extends Animal {
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = !this.hidden;
+  }
+}
+
+class Carnivore extends Animal {
+  bite(animal) {
+    if (animal instanceof Carnivore || animal.hidden) {
+      return;
+    }
+
+    animal.health -= 50;

```

Comment:
```suggestion
    if (animal instanceof Herbivore && !animal.hidden) {
      animal.health -= 50;;
    }
```
Let's get rid of empty `return` and make your code shorter